src/plugins/dashboard/public/dashboard_container/embeddable/create/create_dashboard.ts

```diff
@@ -629,7 +497,8 @@ export const initializeDashboard = async ({
       sessionIdToRestore ??
       (existingSession && incomingEmbeddable ? existingSession : session.start());
 
-    untilDashboardReady().then((container) => {
+    untilDashboardReady().then(async (container) => {
+      await container.untilContainerInitialized();
```

Comment:
Should we comment this out? Probably no harm in keeping it, but 🤷 We are also calling `untilContainerInitialized` in `src/plugins/embeddable/public/lib/containers/container.ts` (technically unnecessarily, until we re-implement the await)